failed: run_tests_grpc-node_macos_opt_native failing in 100% cases

I think it's flaky.
From #12760 
https://kokoro.corp.google.com/job/grpc/job/macos/job/pull_request/job/grpc_basictests_opt/2662/
and #12797 
https://kokoro.corp.google.com/job/grpc/job/macos/job/pull_request/job/grpc_basictests_opt/2664/
```
[16:57:31] Error: Command failed: npm install --build-from-source --unsafe-perm
    at /Volumes/BuildData/tmpfs/src/github/grpc/grpc-node/node_modules/execa/index.js:236:11
    at process._tickCallback (node.js:438:9)

2017-10-05 16:57:31,376 FAILED: tools/run_tests/helper_scripts/run_grpc-node.sh [ret=1, pid=90369]
2017-10-05 16:57:31,377 FAILED: tools/run_tests/helper_scripts/run_grpc-node.sh
2017-10-05 16:57:31,383 FAILED: Some tests failed

2017-10-05 16:57:31,416 FAILED: run_tests_grpc-node_macos_opt_native [ret=2, pid=89555]
2017-10-05 16:57:31,420 START: run_tests_ruby_macos_opt_native
```

This is failing in 100% cases on master.

The actual error message:
```
CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/max_age/max_age_filter.o
  CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/message_size/message_size_filter.o
  CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/workarounds/workaround_cronet_compression_filter.o
  CXX(target) Release/obj.target/grpc/deps/grpc/src/core/ext/filters/workarounds/workaround_utils.o
  CXX(target) Release/obj.target/grpc/deps/grpc/src/core/plugin_registry/grpc_plugin_registry.o
  LIBTOOL-STATIC Release/libgrpc.a
make: *** No rule to make target `Release/obj.target/gpr/deps/grpc/src/core/lib/profiling/basic_timers.o', needed by `Release/libgpr.a'.  Stop.
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/Users/kbuilder/.nvm/versions/node/v4.8.4/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:276:23)
gyp ERR! stack     at emitTwo (events.js:87:13)
gyp ERR! stack     at ChildProcess.emit (events.js:172:7)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:211:12)
gyp ERR! System Darwin 15.6.0
```

https://sponge.corp.google.com/target?id=cfc1dd35-48f2-4717-9e85-4c8daeded791&target=github/grpc/grpc-node_macos_opt_native&searchFor=&show=FAILED&sortBy=STATUS

--- a/templates/tools/dockerfile/test/node_jessie_x64/Dockerfile.template
+++ b/templates/tools/dockerfile/test/node_jessie_x64/Dockerfile.template
@@ -32,5 +32,8 @@
   <%include file="../../python_deps.include"/>
   <%include file="../../node_deps.include"/>
   <%include file="../../run_tests_addons.include"/>
+  # Install Mako to generate files in grpc/grpc-node
+  RUN pip install Mako
+
   # Define the default command.
   CMD ["bash"]

--- a/tools/dockerfile/test/node_jessie_x64/Dockerfile
+++ b/tools/dockerfile/test/node_jessie_x64/Dockerfile
@@ -102,5 +102,8 @@ RUN ln -s /usr/bin/ccache /usr/local/bin/clang++
 
 RUN mkdir /var/local/jenkins
 
+# Install Mako to generate files in grpc/grpc-node
+RUN pip install Mako
+
 # Define the default command.
 CMD ["bash"]

--- a/tools/internal_ci/helper_scripts/prepare_build_macos_rc
+++ b/tools/internal_ci/helper_scripts/prepare_build_macos_rc
@@ -60,7 +60,7 @@ pod repo update  # needed by python
 
 # python
 pip install virtualenv --user python
-pip install -U six tox setuptools twisted pyyaml --user python
+pip install -U Mako six tox setuptools twisted pyyaml --user python
 export PYTHONPATH=/Library/Python/3.4/site-packages
 
 # set xcode version for Obj-C tests

